### PR TITLE
entries: Bung 1st compo

### LINKED ISSUE #183: Bung Enterprises 1st Coding Competition
The entries/results for the first Bung Enterprises Coding Competition can be found here:

http://n64.icequake.net/mirror/www.bung.com.hk/GBCodingCompetition/1stGBCodingCompetition/results.htm

A more complete name of the compo is "1st Amateur GameboyTM Format Games Coding Competition"

### PR DESCRIPTION
**Pull request by dag7dev**
Close #183

### CODE REVIEW

File: game-schema-d4.json [hunk 1]
```diff
@@ -88,31 +88,33 @@
...
+                    "Shooter",
+                    "Simulation",
+                    "Sports",
+                    "Strategy",
+                    "Survival",
+                    "Visual Novel",
                     "gbajam21",
                     "gbcompo21",
                     "gbcompo21-shortlist",
-                    "gbprinter",
```
Comment (avivace) [inline game-schema-d4.json:100]: This file got auto-formatted probably and introduced a bunch of unrelated changes, can you please disable the formatter and only commit the additional tag ?

**Review by avivace — CHANGES_REQUESTED** [review]

File: game-schema-d4.json (hunk 1 above)
Comment (dag7dev) [reply game-schema-d4.json]: Done!

**Review by avivace — CHANGES_REQUESTED** [review]
LGTM. Maybe we could create an entry for "MM2000", the only one missing?

An entry with just the metadata and the GIF that can be found on the compo website but no ROM..

**Comment by dag7dev** [comment]
> LGTM. Maybe we could create an entry for "MM2000", the only one missing?
> 
> An entry with just the metadata and the GIF that can be found on the compo website but no ROM..

Should we create an issue about "Lost homebrews" instead? Probably we're gonna to find out other roms...

File: entries/Wonderworm-Willy/game.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "description": "Willy Wonderworm is a game for the Nintendo Gameboy and Gameboy Color systems. It should operate under most emulators and on an actual Gameboy system if copied onto a cartridge. Note that the scrolling text on the title screen may not appear correctly on some emulators and we recommend that you use the nocash emulator (no$gmb).",
```
Comment (avivace) [inline entries/Wonderworm-Willy/game.json:2]: This entry does not have a working rom...

**Review by avivace — CHANGES_REQUESTED** [review]

File: entries/Wonderworm-Willy/game.json (hunk 2 above)
Comment (dag7dev) [reply entries/Wonderworm-Willy/game.json:2]: It's not my fault, I didn't add it, it doesn't come from me.

However, I notice that it is already on HH under another page, and it perfectly works.
(here: https://hh.gbdev.io/game/willy-wonderworm)

Not sure how to proceed here:
1. remove the Wonderworm-Willy folder, add missing infos to willy-wonderworm (the correct one)
2. don't care about Wonderworm-Willy, add missing infos to willy-wonderworm (the correct one). We will later open a MR that will take care of removing Wonderworm-Willy.

Lemme know.

Comment (avivace) [reply entries/Wonderworm-Willy/game.json:2]: No one has "faults" here.. but if the PR/Issue is about adding the entries from an event, for which a working link was provided, listing every entry with the corresponding ROMs, so a PR adding entries from such event should at least require you to go to every entry and make sure they are in the database. This didn't happen, as you added a tag to a broken entry ..

If I have to go through every diff you introduced to make sure you are applying a tag to the correct entry because you didn't even check what you are applying a tag to...well... it's better for me to restart the PR on my own

Comment (dag7dev) [reply entries/Wonderworm-Willy/game.json]: Done!

**Review by avivace — APPROVED** [review]
